ui/css/treeherder.css

```diff
@@ -1282,6 +1282,71 @@ fieldset[disabled] .btn-unclassified-failures.active {
   border-color: #e0e0e0;
 }
 
+.btn-filter-chicklets {
+  background-color: white;
+  border-color: #a9a9a9;
+  color: white;
+}
+
+.btn-nav-filter {
+  padding: 7px 2px;
+}
+
+.chicklet-off {
+  text-decoration: line-through;
+  opacity: 0.5;
+}
+
+.btn-filter-chicklet-orange,
+.btn-filter-chicklet-orange:hover {
+  color: #dd6602;
+}
+
+.btn-filter-chicklet-red,
+.btn-filter-chicklet-red:hover {
+  color: #C03A44;
```

Comment:
I noticed a mix of uppercase hex color here and a few others (L1312,L1322), vs. the rest lowercase. I read somewhere on a google style guide that lowercase hex was better for readability, allegedly :)